tests/framework/jquery/validators/NumberValidatorJqueryClientScriptTest.php

```
@@ -139,43 +172,76 @@ public function testClientValidateComparison(): void
+        self::assertStringContainsString(
+            '"max":13.37',
+            $js,
+            "Failed asserting that the generated client validation script contains the expected 'max' value.",
+        );
+    }
+
+    public function testClientValidateAttributeWithCustomClientScriptAndUseJqueryFalse(): void
+    {
+        Yii::$app->useJquery = false;
+
+        $modelValidator = new FakedValidationModel();
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$modelValidator'. (undefined)

(UnusedLocalVariable)